Testing/UX/Authentication/AuthUtils.cs

```diff
@@ -6,6 +6,7 @@
 **/
 using System;
 using Dashboard.Client.SessionManagement;
+using Dashboard;
```

Comment:
why changes in UX?

Reply:
This class is implementing a interface in dashboard... And this PR adds a function to that interface... Hence a function is added to avoid build errors